docs/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 ### Added
 - Introduce flaw ownership through task management system (OSIDB-69)
+- Introduce FlawReference and FlawReferenceManager models (OSIDB-71)
```

Comment:
As this PR does not bring anything visible to the users and not something extra large or of some user interest, I would omit mentioning the change in the changelog. The better timing is when there is finally some API.